### PR TITLE
Rsh/adding pipelines

### PR DESCRIPTION
**Pull request by ramsessanchez**
Fixes #

### Changes proposed in this pull request
-

### Other links

-

### CODE REVIEW

**Review by MIchaelMainer — APPROVED** [review]

File: .azure-pipelines/buildAndPackage.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#Licensed under the MIT License.
+#Building and packaging the artifacts of the Java-SDK libraries using the build.gradle file.
+#Ready the package for deployment and release. 
+
+trigger:
+  branches:
+    include:
+    - dev 
+    - main
+    - feature/v3
```
Comment (baywet) [inline .azure-pipelines/buildAndPackage.yml:11]: remove the feature/v3 branch, it's a remain of when I was working on v3

File: .azure-pipelines/buildAndPackage.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      - CONTRIBUTING.md
+      - THIRD PARTY NOTICES
+      - README.md
+
+pr: none
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
```
Comment (baywet) [inline .azure-pipelines/buildAndPackage.yml:30]: This currently only defines the build stage. A multi-stages pipeline should be defined to also contain the CD steps.
It should leverage environments and approvals
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/approvals?view=azure-devops&tabs=check-pass
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/environments?view=azure-devops
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#stage

File: .azure-pipelines/buildAndPackage.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#Copyright (c) Microsoft Corporation. All rights reserved.
+#Licensed under the MIT License.
+#Building and packaging the artifacts of the Java-SDK libraries using the build.gradle file.
+#Ready the package for deployment and release. 
+
+trigger:
+  branches:
+    include:
+    - dev 
+    - main
```
Comment (baywet) [inline .azure-pipelines/buildAndPackage.yml:10]: no main branch on this repo, remove

File: .azure-pipelines/buildAndPackage.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      - docs/*
+      - .github/*
+      - LICENSE
+      - .gitignore
+      - README-localized/*
+      - CONTRIBUTING.md
+      - THIRD PARTY NOTICES
+      - README.md
+
+pr: none
```
Comment (baywet) [inline .azure-pipelines/buildAndPackage.yml:25]: This should probably trigger on PRs as well to provide feedback.

**Pull request merged**